src/components/QuizGameQuestions/index.jsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import "./index.css";
+
+// this function will display the fetched data to the dom.those parameters will pass in quizgame component.
+// the callback is used to check if the answer is true or not.
+// types of props ={
+//  question: string
+//  storedQuestionsAnswers string
+//  callback function
+//  userAnswer string 
+//  questionNum number
+//  totalQuestions number
+//}
+
+const QuestionCard = ({
+  question,
+  storedQuestionsAnswers,
```

Comment:
i don't understand this variable. is it the answers for the questions? storedQuestionsAnswers has plural Questions, so it sounds like it is multiple questions and multiple answers

if its the answers for the questions, then call it answerChoices

Reply:
PTAL